pmultiqc/modules/quantms/quantms.py

```diff
@@ -129,7 +128,8 @@ def __init__(self):
         self.draw_precursor_charge_distribution()
         self.draw_peaks_per_ms2()
         self.draw_peak_intensity_distribution()
-        self.draw_delta_mass()
+        self.draw_oversampling()
+        # self.draw_delta_mass()
```

Comment:
why did this go?

Reply:
Because when there is a lot of psm data for delta masss, the browser needs to draw many points, which greatly reduces its speed.

Reply:
Can we maybe plot a histogram of delta masses? Then we could decide on the number of bins and update them on-the-fly, without storing all deltas.

Reply:
But we can do that in another PR. Let's merge and test in the workflow?

Reply:
ok.  I can try to do that in another PR.